makefile

```diff
@@ -59,29 +56,31 @@ BUILD_DEFINES:= \
 	-D__BUILD_VERSION__=\""$(CONFIG_VERSION)"\"
 
 # --- Configs -------------------------------------------- #
-CC:=i686-pc-skift-gcc
+CC:=gcc
```

Comment:
why

Reply:
I didn't mean for my changes to the makefile to be included into the pull request. Must of happened by accident. Oops.